tests/DCPS/LargeSample/run_test.pl

```diff
@@ -30,8 +30,8 @@
     $sub_opts .= "-DCPSConfigFile multicast.ini ";
 }
 
-my($pub1opts, $pub2opts) =
-    $PerlDDS::SafetyProfile ? ('-p 1', '-p 2') : ('' , '');
+my($pub1opts) =
+    $PerlDDS::SafetyProfile ? ('-p 1') : ('');
 
 $pub_opts .= " -r $reliable -n $num_msgs";
 $sub_opts .= " -r $reliable -n " . ($num_msgs * 4);
```

Comment:
```suggestion
$sub_opts .= " -r $reliable -n $num_msgs";
```
Since you've removed the second publisher and the second writer per publisher, the expected number of messages per subscriber goes from 40 to 10 (which is why the tests were "failing").

Reply:
You are completely right. I totally forgot to remove the the multiplication with 4